src/HeckeTypes.jl

```diff
@@ -544,12 +544,11 @@ mutable struct FacElemMon{S} <: Ring
 
   function FacElemMon{S}(R::S, cached::Bool = false) where {S}
     return get_cached!(FacElemMonDict, R, cached) do
-      z = new()
-      z.base_ring = R
-      z.basis_conjugates_log = Dict{RingElem, Vector{arb}}()
-      z.basis_conjugates = Dict{RingElem, Vector{arb}}()
-      z.conj_log_cache = Dict{Int, Dict{nf_elem, arb}}()
-      return z
+      new{AnticNumberField}(R,
```

Comment:
```suggestion
      new{S}(R,
```

Reply:
Indeed, stupid copy&paste. Thank you for catching this early